docs: :memo: edit interface index, inputs, and outputs page (#1399)
seedcase-project/seedcase-sprout: docs/design/interface/outputs.qmd

:   This machine-readable JSON file is the foundation of a data package
    and contains the full [Frictionless Data
    specification](https://datapackage.org/). This file contains
    metadata on the package (like title, description, and contributors)
    as well as metadata on the data resources (like column names, data
    types, descriptions, and a path to the Parquet file with the data in
    it).

`./README.md`

> :   The `README.md` file is an auto-generated, more human readable

Reviewer: Good point. It isn't "auto-" generated, just generated :+1:

> :   The `README.md` file is an auto-generated, more human readable

Reviewer: ```suggestion
:   The `README.md` file is a generated, more human readable
```
